src/site/content/en/progressive-web-apps/promote-install/index.md

```diff
@@ -44,31 +46,40 @@ using on your site.
   completed a purchase.
 * Combine more than one of these techniques in different parts of your PWA,
   but be careful not to overwhelm or annoy your user with install promotion.
-* Only show the promotion when you detect that the `beforeinstallprompt`
-  event has been fired.
+* Only show the promotion **after** the
+  [`beforeinstallprompt` event](/customize-install/#beforeinstallprompt) has
+  been fired.
 
 ## Automatic browser promotion {: #browser-promotion }
 
-<figure class="w-figure w-figure--inline-right">
-  <img src="pwa-omnibox-install.jpg" alt="Install button in the omnibox for Hulu.com">
-  <figcaption class="w-figcaption">
-    Install button in the omnibox provided by the browser.
-  </figcaption>
-</figure>
-
-When [certain criteria](/install-criteria/) are met, the browser will
+When [certain criteria](/install-criteria/) are met, most browsers will
 automatically indicate to the user that your Progressive Web App is
 installable. For example, desktop Chrome shows an install button in the omnibox.
 
+<div class="w-columns">
+  <figure class="w-figure" id="browser-install-promo">
+    <img src="how-does-it-work_desktop.png"
+        alt="Screenshot of omnibox with install indicator visible.">
+    <figcaption class="w-figcaption">
+      Browser provided install promotion (desktop)
+    </figcaption>
+  </figure>
+  <figure class="w-figure">
+    <img src="how-does-it-work_mobile.png"
+        alt="Screenshot of browser provided install promotion.">
+    <figcaption class="w-figcaption">
+      Browser provided install promotion (mobile)
+    </figcaption>
+  </figure>
+</div>
```

Comment:
@robdodson these images have also been used in another post that @petele is updating. Pete is currently duplicating the images in the `/install-criteria` directory. Would you prefer to use a single source of the images?

Reply:
I think I'm ok with either approach.

- If we update the image in /install-criteria then this post will receive the update (yay!)
- However, if we delete the image in /install-criteria then we have to remember to delete it here (boo)
- If we duplicate the images then we have to remember to update them in both places but it does mean that removing post A will not have any negative effect on post B.

Reply:
I was also thinking that it was fine to duplicate because of the risk that we update one article and forget that the other also depends on it. I just wanted to make sure I wasn't missing anything from an engineering perspective.